corehq/apps/users/templates/users/mobile_workers.html

```diff
@@ -359,10 +361,17 @@ <h3 class="panel-title">{% trans 'Mobile Workers' %}</h3>
                   </p>
                 </div>
                 <div data-bind="visible: !is_active() && !is_account_confirmed()">
+                  {% if request|toggle_enabled:"TWO_STAGE_USER_PROVISIONING" %}
```

Comment:
Could this be `{% if two_stage_user_confirmation %}`?

Reply:
This is having if-else condition for email or sms components, so adding specific flag check.

Reply:
got it, I misread the flag names